.github/dependabot.yml

```
@@ -31,3 +31,7 @@ updates:
         versions: ['>7.3.3']
       - dependency-name: '@types/inquirer'
         versions: ['>7.3.1']
+  - package-ecosystem: 'github-actions'
+    directory: /
```

This is a really small nitpick, but I'm going to add quotes around this value here:
```suggestion
    directory: '/'
```